tenants-certified-attributes-updater/src/main/scala/it/pagopa/interop/tenantscertifiedattributesupdater/util/package.scala

```diff
@@ -81,7 +81,16 @@ package object util {
         }
         .groupMapReduce[ExternalId, List[InternalAttributeSeed]](_._1)(_._2)(_ ++ _)
         .toList
-        .map(Function.tupled(InternalTenantSeed))
+        .map(x =>
+          InternalTenantSeed(
+            x._1,
+            x._2,
+            tenants.find(t => t.externalId == PersistentExternalId(x._1.origin, x._1.value)) match {
+              case Some(pTenant) => pTenant.name
+              case None          => ""
+            }
+          )
+        )
```

Comment:
To boost performances you can prepare a map like 
```
val tenantsMap: Map[PersistentExternalId, String] = tenants.map(t => (t.externalId, t.name)).toMap
```
and then
```suggestion
        .map{ case (extId, attrs) =>
          InternalTenantSeed(
            extId,
            attrs,
            tenantsMap.getOrElse(PersistentExternalId(extId.origin, extId.value), "")
          )
        }
```